### ISSUE TITLE
Link to member's github profile if they don't have a website

### ISSUE THREAD

**Issue by danieltott** [Virtual-Coffee/virtualcoffee.io#69]
## Issue Context

Currently on the [members page](https://virtualcoffee.io/members/), if a user doesn't have a website in their github profile, their name doesn't get a link. Let's set this to link to their github profile instead, so all the members' names look the same.

## Suggested Solution

In `src/members.njk`, we display their name like this:

```njk
{% if member.blog %}
  <a href="{{member.blog}}" target="_blank" rel="noopener noreferrer">{{member.name}}</a>
{% else %}
  {{member.name}}
{% endif %}
```

Let's use a link to their github profile (`member.html_url`) instead of just the plain name if there's no `member.blog`

This will need to be done for `founders` and `communityEngagement` as well.

**Comment by surajrpanchal** [comment]
I’ll handle this one

**Comment by danieltott** [comment]
Fixed in #71
Thank you @surajrpanchal!!

![fist bump](https://media3.giphy.com/media/Dnt2VnWFknFNm/giphy.gif)
